src/main/java/com/minecolonies/rankup/util/RankingUtils.java

```diff
@@ -13,6 +13,11 @@
 public class RankingUtils
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this class "final" or add a public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2974)